foolbox/attacks/localsearch.py

```diff
@@ -149,8 +149,8 @@ def score(It):
             # Update a neighborhood of pixel locations for the next round
             PxPy = [
                 (x, y)
-                for a, b in PxPy_star
-                for x in range(a - d, a + d + 1)
-                for y in range(b - d, b + d + 1)]
+                for _a, _b in PxPy_star
```

Comment:
yep, because in Python2 the "a" in the for-loop was overwriting the adversarial object.